sceptre/plan/actions.py

```diff
@@ -1000,3 +1000,147 @@ def diff(self, stack_differ):
         :rtype: sceptre.diffing.stack_differ.StackDiff
         """
         return stack_differ.diff(self)
+
+    @add_stack_hooks
+    def drift_detect(self) -> Tuple[str, dict]:
```

Comment:
Depending on the situation this method might not return a `Tuple[str, dict]`. In fact, in its current form, it will usually return a Tuple[str, str]`